Products/ZenModel/UserInterfaceSettings.py

```diff
@@ -34,6 +34,7 @@ class UserInterfaceSettings(ZenModelRM):
         {'id': 'showEventSeverityIcons', 'type': 'boolean', 'mode': 'w'},
         {'id': 'enableTreeFilters', 'type': 'boolean', 'mode': 'w'},
         {'id': 'showPageStatistics', 'type' : 'boolean', 'mode':'w'},
+        {'id': 'renderNetworksFromCatalog', 'type' : 'boolean', 'mode':'w'},
```

Comment:
I'm not sure why this should be a user configurable option. If using the catalog is more performant then it should always be used. Under what circumstances would a user prefer one versus the other?